entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -16,6 +16,103 @@ The plan describes the overall themes for EF Core 5.0, including everything we a
 
 We will add links from here to the official documentation as it is published.
 
+## Preview 5
+
+### Database collations
+
+The collation to use for database columns can now be specified when building the model and will flow through to generated migrations.
```

Comment:
Is it more interesting/relevant to show the database-wide collation? This is fine too, no strong feelings.

Reply:
I'll add a database-wise example as well.